src/Packetery/Core/Api/InvalidRequestException.php

```diff
@@ -16,4 +16,31 @@
  */
 class InvalidRequestException extends \Exception {
 
+	/**
+	 * Error messages.
+	 *
+	 * @var array
+	 */
+	private $messages;
+
+	/**
+	 * InvalidRequestException.
+	 *
+	 * @param string $mainMessage Main message.
+	 * @param array  $errorMessages Array of messages.
```

Comment:
Chceš připomínat zarovnávání komentářů? A nebo to už nechceš dělat? Obecně bych to nechat na automatu a nemyslel bych na to.